projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -106,7 +106,18 @@ public static function is_wpcom() {
 	 */
 	public static function assert_is_wpcom( $method ) {
 		if ( ! self::is_wpcom() ) {
-			throw new \Exception( esc_html( "Method $method can only be called on WordPress.com." ) );
+			throw new \Exception( esc_html( "Method $method can only b`e called on WordPress.com." ) );
 		}
 	}
+
+	/**
+	 * Check if the new module endpoint is available in the used Jetpack version.
+	 * We need the module status in response that's why we do the version check https://github.com/Automattic/jetpack/pull/41461/files#diff-f8e5ef1115599de750b64143dd1901554254eddd95ab4371b6b6b3b2a5914224R638-R642.
+	 * More: https://github.com/Automattic/jetpack-reach/issues/794
```

Comment:
Since that is a private repo, may be we should mention the PR here?

```suggestion
	 * More: https://github.com/Automattic/jetpack/pull/41596
```